dio/lib/src/sentry_dio_extension.dart

```diff
@@ -15,9 +15,7 @@ extension SentryDioExtension on Dio {
   @experimental
   void addSentry({
     bool recordBreadcrumbs = true,
```

Comment:
Could be a new option in SentryOptions as well since it's being used by this integration and also `SentryHttpClient`.
Maybe called `recordHttpBreadcrumbs`